static/js/settings_org.js

```diff
@@ -844,24 +850,14 @@ exports.build_page = function () {
 
     $("#id_realm_org_join_restrictions").change(function (e) {
         const org_join_restrictions = e.target.value;
-        const node = $("#allowed_domains_label").parent();
+        const node = $("#zero_domains_warning").parent();
         if (org_join_restrictions === 'only_selected_domain') {
```

Comment:
This is the third time I'm seeing this check for showing/hiding the warning. Maybe this can be extracted to a small `show_hide_domain_warning(org_join_restrictions);`.

Reply:
I removed this whole block in the next commit introducing "Domain restrictions" section, it's still good to have a function for this, will add a function then.

Reply:
Sorry, I was reviewing this commit by commit. For 2 cases, it doesn't seem that important.

Reply:
Okay, didn't change it.